test-scripts/hit_endpoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+curl -X POST localhost:8090/sign -H "Content-Type: application/json" -d "{\"message\": \"test\", \"sigValidityOverride\": 0}"
```

Comment:
Do we need to document this script somewhere to show what it's for?

Reply:
A test-scripts README has been added with documentation for this script.